pkgs/games/runescape-launcher/default.nix

```
@@ -103,6 +103,13 @@ in
     ];
     multiPkgs = pkgs: [ libGL ];
     runScript = "runescape-launcher";
+    extraInstallCommands = ''
+      mkdir -p "$out/share/applications"
+      cp ${runescape}/share/applications/runescape-launcher.desktop "$out/share/applications"
+      cp -r ${runescape}/share/icons "$out/share/icons"
+      substituteInPlace "$out/share/applications/runescape-launcher.desktop" \
+        --replace "/usr/bin/runescape-launcher" "$out/bin/RuneScape"
```

```suggestion
        --replace "/usr/bin/runescape-launcher" "RuneScape"
```

Using a full path will confuse some desktop environments when a future update changes it.

---

Thanks, updated the PR.